### DISCUSSION TITLE
Brainstorm!

### DISCUSSION THREAD

**Discussion by jeremymanning** [ContextLab/chatify#14] [Ideas]
Add your brainstormed ideas here!  ⛈️

**Comment by AGuyNextDoor** [comment]
## V2 ideas
## IDEA :
Create a "fine-tuning" with the text embeddings of the Neuromatch curriculum

## MOTIVATION : 
- Students can asks a specific question on the curriculum
- Reinforce the knowledge validity of the LLM

## HOW : 
- Create a VectorStore with the embeddings of the different curriculums (or the complete curriculum)
- Actually fine-tune an LLM model (creates a lot of complications)

## GOING FURTHER : 
- One embedding for every different type of curriculum?
